modules/apps/object/object-test/src/testFunctional/tests/ObjectWorkflow.testcase

```diff
@@ -23,12 +23,53 @@ definition {
 	}
 
 	@description = "LPS-145228 - Verify that the user is able to preview the entry's information inside of the My Workflow Tasks"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanPreviewEntryInformationOnMyWorkflowTasks {
 		property portal.acceptance = "true";
 
-		// TODO LPS-146873 CanPreviewEntryInformationOnMyWorkflowTasks pending implementation
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.addObjectFieldViaAPI(
+			fieldLabelName = "Custom Field",
+			fieldName = "customObjectField",
+			fieldType = "String",
+			isRequired = "false",
+			objectName = "CustomObject");
```

Comment:
Hi @andreefarias 

You can add the parameter `fieldBusinessType = "Text"` ? 

thanks